_includes/master/reqs-azure.md

```diff
@@ -0,0 +1,46 @@
+## About {{site.prodname}} on Azure
+
+Azure does not support {{site.prodname}} networking. Use one of the following
```

Comment:
Too bald?  Perhaps insert "natively" ?  i.e. "Azure does not natively support ..."

Reply:
I might spin it as "While Azure does not have support Calico networking, Calico policy is commonly used on Azure in conjunction with one of the following networking options"